common-pg/src/main/scala/com/socrata/pg/analyzer2/SoQLRep.scala

```diff
@@ -872,6 +872,94 @@ abstract class SoQLRepProvider[MT <: MetaTypes with metatypes.SoQLMetaTypesExt w
         }
     },
 
+    SoQLBlob -> new SingleColumnRep(SoQLBlob, d"text") {
```

Comment:
This "text" is the SQL type backing the column

Reply:
thanks, I was getting tripped up comparing this to SoQLDocument. Though I guess I thought Document and Photo were both underlyingly blob type?

Reply:
Yeah, they're not - blob and photo are both just pointers into the core blobstore.  Document is that plus some metadata.

Reply:
blob and photo both potentially have additional metadata columns, though right? Like blobs can have a name and Photos can have a name and a content_type
https://cate-scgc.test-socrata.com/resource/22g9-33r8.json

Reply:
(exploring around I see that content_type and filename are attributes of Document, and the other two are simple, like Robert's saying. Cool!